compiler/inferring/node.h

```diff
@@ -30,26 +29,37 @@ class Node : public Lockable {
   // using forward_list here is a bit-bit slower than vector, but takes considerably less memory
   // these two lists are separate variables, as joining them into one list causes negative performance impact
 
-  volatile int recalc_state_;
+protected:
+  enum {
+    recalc_st_waiting = 0,
+    recalc_st_processing = 1,
+    recalc_st_need_relaunch = 2,
+    // and three more states — the same by meaning, but with bit 16 on (16, 17, 18)
+    // that 3 more states — with bit 16 on — mean that recalc has once finished <=> recalc count > 0
+    recalc_bit_16 = 16,
```

Comment:
I don't like name of this enum value. Maybe recalc_bit_at_least_once?

Reply:
Ok